src/palakSJun20/FibonacciNumbers.java

```diff
@@ -0,0 +1,32 @@
+/*Program 2: To print first n Fibonacci Numbers.
+Input n = 5
+Output = 1 1 2  3  5  8 
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55*/
+package palakSJun20;
+
+public class FibonacciNumbers {
+
```

Comment:
add method  in comments like://method to print to Fibonacci Series

Reply:
Done